### DISCUSSION TITLE
Github Octoverse Form

### DISCUSSION THREAD

**Discussion by Aryankpoor** [github/india#117] [Satellite India 2021 Q&A] [answered]
The Github octoverse form is too long. A form is supposed to be short and quick for the user. I personally lost interest after 10 questions. When a user clicks on the form its supposed to be filled when the user has some leftover time . Instead the form is like that you have to dedicate some time or take out a particular instance of time , sit down and then fill the form. Its like I am going through government paperwork. This is what I personally fell. If anyone else also feels that way or I raised this point in the wrong section I don't know.

**Comment by FRTMentor** [comment, marked as answer]
Please reach out the GitHub support team.
